src/tasks/Directory.Build.props

```diff
@@ -3,5 +3,6 @@
 
   <PropertyGroup>
     <TargetFrameworkForNETFramework>net472</TargetFrameworkForNETFramework>
+    <TargetFrameworkForNetCore>net6.0</TargetFrameworkForNetCore>
```

Comment:
In general, what criteria will push us to update this value?

Reply:
Nit: `TargetFrameworkForNETCore`?

Reply:
Any new features we may need.  We may not need to bump all that often.